nrich-search/README.md

```diff
@@ -146,7 +156,7 @@ public class CarSearchResult {
 
 Then in our service we would invoke it like this:
 
-```
+```java
```

Comment:
Maybe rename service class to `CarSearchService`. I would omit `Demo` here since all other classed are without it.

Additionally, maybe reformat (define in multiple lines) build of search configuration in `search` and `simpleSearch` method to improve readability.